docs/src/components/demo/Demo.tsx

```diff
@@ -19,14 +19,12 @@ import { StackBlitzLink } from './StackBlitzLink';
 import classes from './Demo.module.css';
 
 export interface DemoProps {
-  componentName: string;
-  demoName: string;
   variants: BaseDemo.DemoVariant[];
   defaultCodeOpen?: boolean;
 }
 
 export function Demo(props: DemoProps) {
-  const { componentName, demoName, variants: demoVariants, defaultCodeOpen = true } = props;
```

Comment:
imo not worth it to require a component name prop just so we can display a different CodeSandbox title, seems like unnecessary legwork for us